### PR TITLE
Recipe for coq-jupyter

### PR DESCRIPTION
**Pull request by slel**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by slel** [comment]
cc @isuruf

File: recipes/coq-jupyter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/coq_jupyter-{{ version }}.tar.gz
+  sha256: c8bee708c15bfa6d80c1c34e04617b794ce2b2dde4478af7679fa21ed92b15f5
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (scopatz) [inline recipes/coq-jupyter/meta.yaml]: Can this be `noarch: python`?

Comment (isuruf) [reply recipes/coq-jupyter/meta.yaml]: There's a post-link.sh script that activates the jupyter extension.

Comment (scopatz) [reply recipes/coq-jupyter/meta.yaml]: Ahh ok

**Comment by isuruf** [comment]
Ready for review

**Pull request merged**